skimage/measure/profile.py

```diff
@@ -44,11 +45,16 @@ def profile_line(img, src, dst, linewidth=1,
            [0, 0, 0, 0, 0, 0]])
     >>> profile_line(img, (2, 1), (2, 4))
     array([ 1.,  1.,  2.,  2.])
+    >>> profile_line(img, (1, 0), (1, 6), cval=4)
+    array([ 1.,  1.,  1.,  2.,  2.,  2.,  4.])
 
-    Notes
-    -----
     The destination point is included in the profile, in contrast to
     standard numpy indexing.
+    For example: 
+    >>> profile_line(img, (1, 0), (1, 6)) # The final point is out of bounds
```

Comment:
One last small and very nitpicky change:  Python PEP8 requires two spaces before the "#" comment character.